components/git/staging.js

```diff
@@ -0,0 +1,67 @@
+import CLI from '../../lib/cli.js';
+import { runPromise } from '../../lib/run.js';
+import { Staging } from '../../lib/staging.js';
+
+export const command = 'staging';
+export const describe = 'Automatic port commits to a release line branch';
+
+const stagingOptions = {
+  backport: {
+    describe: 'The PR ID / number to backport, skip staging commits',
```

Comment:
I don't understand why this option is on `git node staging`?

Reply:
The idea is that this is a convenience helper for when, let's say, you end up having to remove a commit from the staging branch after figuring out it was breaking CI, then as a releaser you can just run `git node staging --backport=<PR_ID>` after rebasing / removing the offending commits from the staging branch.

It's part of `git node staging` more from an implementation convenience point of view since all the internal logic is already there.

Reply:
Also, it's worth noting that `git node staging --backport=<PR_ID>` should send an automated comment by default, which it's not going to happen in the regular `git node staging` automation.